squidb-addons/squidb-json/squidb-json-plugin/src/com/yahoo/squidb/json/JSONPropertySupport.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Copyrights licensed under the Apache 2.0 License.
+ * See the accompanying LICENSE file for terms.
+ */
+package com.yahoo.squidb.json;
+
+import com.yahoo.squidb.data.AbstractModel;
+import com.yahoo.squidb.utility.Logger;
+
+import java.lang.reflect.Type;
+
+/**
+ * Business logic of managing serialization and deserialization of JSON properties. Clients of this plugin should
+ * be sure to call {@link #setJSONMapper(JSONMapper)} to initialize the serialization engine. See the JSONPropertyTest
+ * class in the squidb tests project for example implementations of the JSONMapper interface.
+ */
+public class JSONPropertySupport {
+
+    private static final String TAG = "squidb-json";
+
+    public static JSONMapper MAPPER;
```

Comment:
Probably should be private so we could add some kind of defensive code in `setJSONMapper()`